ui/components/app/account-menu/account-menu.component.js

```diff
@@ -413,6 +471,75 @@ export default class AccountMenu extends Component {
           }
           text={t('connectHardwareWallet')}
         />
+        {
+          ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
+          <>
+            <AccountMenuItem
+              onClick={() => {
+                toggleAccountMenu();
+                trackEvent({
+                  category: 'Main Menu',
+                  event: 'Clicked Connect custodial account',
+                });
+                if (getEnvironmentType() === ENVIRONMENT_TYPE_POPUP) {
+                  global.platform.openExtensionInBrowser(CUSTODY_ACCOUNT_ROUTE);
```

Comment:
This code seems to be adding another account menu item in addition to the ones already rendered. Is that correct? Just as a sanity check these trackEvents are just duplicated from the existing AccountMenuItem?

Reply:
For the upcoming redesign, you'll also want to add the new item to:

`ui/components/multichain/account-list-menu/account-list-menu.js`

Reply:
Yes, we are adding 3 entries 👍🏼 the trackEvents are specific for MMI, each new menu entry has a different "event", like "clicked connect custodial...", "clicked portfolio", etc

Reply:
Have these new events been vetted against the event schema and @worldlyjohn ?

Reply:
Hum... no. Didn't know it was needed, we've been adding the `trackEvent` events we have in MMI since the beginning 😅

Reply:
thanks for flagging @brad-decker .  

@zone-live it might be helpful to take a look at our [Event Naming Conventions](https://www.notion.so/Event-Naming-Conventions-95bf27517033451888053f7ab654d327).

And as Brad mentioned, events get validated against our schema definitions.  You can open a PR for any event changes/additions here: https://github.com/ConsenSys/segment-schema